### PR TITLE
C++ basic codes

### PR DESCRIPTION
**Pull request by sumitvajarinkar**

### CODE REVIEW

**Review by amanv8060 — CHANGES_REQUESTED** [review]
Change the name of files so that it can be easily understood

**Comment by amanv8060** [comment]
I meant the names of files

File: cpp/Untitled1.cpp [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// it makes difference while using 9 or 9.0 due to type casting.
```
Comment (amanv8060) [inline cpp/Untitled1.cpp:1]: For eg change the name of this file from untitled1.cpp to something that defines functionality of the code

Comment (mikey017) [reply cpp/Untitled1.cpp:1]: Temperature

**Comment by amanv8060** [comment]
As there are no changes being done , i am closing this .

**Pull request closed without merging**